docs/loading-in-profiles.md

```diff
@@ -0,0 +1,45 @@
+# Loading in profiles from various sources
+
+Profiles can be loaded in perf.html from many different sources.
+
+### Online Storage
+
+> `https://perf-html.io/public/{HASH}`
+
+Profiles can be stored in online data store. The hash is used to retrieve it. This is where profiles go when clicking the "Share..." button. Here is an example bash script to programmatically upload profiles:
+
+```bash
+uploadprofile() {
+  gzip $1
+  curl 'https://profile-store.appspot.com/compressed-store' --compressed --data-binary @$1.gz | awk '{print "Hosted at: https://perf-html.io/public/"$1}'
+}
+
+# Execute with the following command:
+uploadprofile /path/to/profile.js
+```
+
+### URL
+
+> `https://perf-html.io/from-url/{URL}`
+
+Profiles can also be loaded in from arbitrary URLs. See the function below for an easy utility for converting to the proper URL format. Here is a simple utility function to turn a profile URL into a perf.html URL to view it:
+
+```js
+function getPerfHtmlUrl (profileUrl) {
+  return `https://perf-html.io/from-url/${encodeURIComponent(profileUrl)}`;
+}
+```
```

Comment:
nit: add something like "You can then upload it to the online storage directly from perf.html's interface"